packages/realm/binding/android/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 cmake_minimum_required(VERSION 3.18.1)
 
-set(REACT_NATIVE_VERSION "0.71.0-rc.5")
```

Comment:
I believe we could simplify by leveragin ggit`find_package`, `find_library` to avoid the entire "Extract .so files from the React Native AAR" block below.

To my knowledge, we currently need `jsi` and `fbjni` headers. We could draw some inspiration from https://github.com/facebook/react-native/blob/main/packages/react-native/ReactAndroid/src/main/jni/CMakeLists.txt#L43-L46.

The React Native packages can refer to these via `target_link_libraries`, see https://github.com/facebook/react-native/blob/main/packages/react-native/ReactAndroid/src/main/jni/react/runtime/hermes/jni/CMakeLists.txt#L27